ci/build_container/build_recipes/zlib.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+VERSION=1.2.11
+
+wget -O zlib-$VERSION.tar.gz https://github.com/madler/zlib/archive/v$VERSION.tar.gz
+tar xf zlib-$VERSION.tar.gz
+cd zlib-$VERSION
+./configure --prefix=$THIRDPARTY_BUILD
+make V=1 install
```

Comment:
FYI I asked @gsagula to add V=1. We have this everywhere else in the recipes, mainly so that it is easy to look at logs and make sure we are compiling with the right options on all the deps.